src/modules/package.ts

```diff
@@ -116,10 +116,15 @@ export class PackageConfig {
     }
 
     isPackageInstalled(): boolean {
-        if (!CliFileSystem.existsSync(this.getPackageDirectoryWithVersion())) {
-            return false;
+        return CliFileSystem.existsSync(this.getPackageDirectoryWithVersion());
```

Comment:
Any reason `isPackageValidRepo`'s logic cannot replace `isPackageInstalled`'s logic? 

The git validity check of the directory should also fail if the directory does not exist.

Reply:
would be possible but that would mean a bit longer rat tail of changes throughout the code because "isPackageInstalled" is synchronous and "isPackageValidRepo" is asynchronous because of the git calls. Shall i still do this?

Reply:
as discussed. rat tail is too long